data-transfer-service/src/main/resources/application.properties

```diff
@@ -69,3 +69,10 @@ dts.listing.listCommand=%s --path '%s' --offset '%d' --size '%d'
 dts.impersonation.enabled=true
 dts.impersonation.name.transformation=LOWERCASE
 dts.impersonation.name.metadata.key=dts_name
+
+
+dts.local.name=${DTS_LOCAL_NAME:}
+dts.local.api.url=${CP_API_URL:}
+dts.local.api.token=${CP_API_JWT_TOKEN:}
+dts.local.preferences.poll=${DTS_LOCAL_POLL_TIMEOUT:30000}
+dts.local.preference.shutdown.key=${DTS_SHUTDOWN_KEY:dts.local.restart}
```

Comment:
Let's get rid of `.local` part in preference names here.

Reply:
Done